webui/react/src/utils/types.ts

```diff
@@ -48,6 +51,7 @@ export const experimentToTask = (experiment: Experiment): RecentTask => {
     lastEvent,
     ownerId: experiment.ownerId,
     progress: typeof experiment.progress === 'number' ? experiment.progress : undefined,
+    startTime: experiment.startTime,
```

Comment:
question: `endTime`? and also `duration` if `endTime` is available?

Reply:
https://github.com/determined-ai/determined/pull/652#discussion_r440493739

Reply:
yep, let's nix duration.